app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+
+import requests
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
You need to generate API_KEY for using Weather API, but it shouldn't be hard-coded in main.py.